custom_components/hyxi_cloud/__init__.py

```
@@ -375,7 +375,134 @@ async def _async_resolve_webhook_url(
+        return
+
+    _LOGGER.debug(
+        "%s: Cancelling prior/orphaned subscription (code: %s)",
+        log_prefix,
+        prior_code,
+    )
+    try:
+        await async_cancel_and_unregister_subscription(
+            hass, coordinator.client, prior_code
+        )
+    except Exception as err:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)